src/rt/sched/threadpool.h

```diff
@@ -82,13 +83,24 @@ namespace verona::rt
 
     ThreadState state;
 
+    /// Pool of cores shared by the scheduler threads.
+    CorePool<ThreadPool<T, E>, E>* core_pool = nullptr;
+
+    /// Systematic ids.
+    size_t systematic_ids = 0;
```

Comment:
Does this need to be `atomic`?  The ids might appear in logs even if systematic testing is not running.